.github/workflows/pull.yml

```diff
@@ -35,9 +35,6 @@ jobs:
     name: test-setup-linux-gcc
     uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
     strategy:
```

Comment:
I think you could just remove this `strategy` section, it does nothing now

Reply:
There is a `fail-fast: false` below